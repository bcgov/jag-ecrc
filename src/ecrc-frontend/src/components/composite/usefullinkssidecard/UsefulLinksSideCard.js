import React from "react";
import SideCard from "../../base/sideCard/SideCard";

export default function UsefulLinksSideCard() {
  return <SideCard />;
}
