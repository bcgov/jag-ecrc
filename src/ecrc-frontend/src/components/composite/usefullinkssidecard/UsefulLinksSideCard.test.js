import React from "react";
import { create } from "react-test-renderer";

import UsefulLinksSideCard from "./UsefulLinksSideCard";

describe("UsefulLinksSideCard Component", () => {
  test("Matches the snapshot", () => {
    const links = [
      {
        name: "Home",
        url: "/"
      },
      {
        name: "Somewhere",
        url: "/somewhere"
      },
      {
        name: "Somewhere else",
        url: "/somewhereelse"
      },
      {
        name: "Somewhere with a long name for no reason",
        url: "/here"
      }
    ];

    const usefulLinks = create(<UsefulLinksSideCard sideCardLinks={links} />);
    expect(usefulLinks.toJSON()).toMatchSnapshot();
  });
});
