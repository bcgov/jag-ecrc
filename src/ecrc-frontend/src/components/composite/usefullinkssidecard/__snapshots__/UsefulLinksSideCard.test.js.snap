// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsefulLinksSideCard Component Matches the snapshot 1`] = `
<div
  className="dashboard-spacing"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col-12"
      style={
        Object {
          "paddingTop": "30px",
          "position": "relative",
        }
      }
    >
      <p />
      <section
        className="submit-container"
        id="blue-section"
      >
        <h2
          className="heading-style"
        >
          Useful Links
        </h2>
        <div
          className="submit-content"
        >
          <p>
            <li
              className="menuItem"
            >
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li
              className="menuItem"
            >
              <a
                href="/somewhere"
              >
                Somewhere
              </a>
            </li>
            <li
              className="menuItem"
            >
              <a
                href="/somewhereelse"
              >
                Somewhere else
              </a>
            </li>
            <li
              className="menuItem"
            >
              <a
                href="/here"
              >
                Somewhere with a long name for no reason
              </a>
            </li>
          </p>
        </div>
      </section>
    </div>
  </div>
</div>
`;
